src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -17,6 +17,8 @@
 @condition_handler_registry.register(Condition.EVENT_FREQUENCY_COUNT)
 @condition_handler_registry.register(Condition.EVENT_UNIQUE_USER_FREQUENCY_COUNT)
 class EventFrequencyCountHandler(DataConditionHandler[list[int]]):
+    type = [DataConditionHandler.Type.WORKFLOW_TRIGGER, DataConditionHandler.Type.ACTION_FILTER]
+
```

Comment:
note for others reading: we're eventually moving event frequency conditions from where they live currently as workflow triggers to action filters